gbpservice/nfp/scripts/mode_shift_enterprise.sh

```diff
@@ -180,29 +191,33 @@ function nfp_logs_forword {
 }
 
 function restart_processes {
-    source $DEVSTACK_DIR/functions-common
-    source $DEVSTACK_DIR/openrc neutron service
-    
-    # restart proxy
-    stop_process proxy
-    run_process proxy "source $NFPSERVICE_DIR/devstack/lib/nfp;namespace_delete $DEVSTACK_DIR;namespace_create $DEVSTACK_DIR $IpAddr"
-    echo "Restarted proxy process"
-    sleep 10
-
-    # restart proxy agent
-    stop_process proxy_agent
-    run_process proxy_agent "sudo /usr/bin/nfp --config-file /etc/nfp_proxy_agent.ini --log-file /opt/stack/logs/nfp_proxy_agent.log"
-    echo "Restarted proxy agent process"
-    sleep 3
+    # restart configurator
+    configurator_ip=120.0.0.3
+    sudo ip netns exec nfp-proxy\
+ ssh -o "StrictHostKeyChecking no" -i configurator_vm root@$configurator_ip\
+ docker exec configurator screen -S configurator -X quit
```

Comment:
We restart the configurator to let it use the modified nfp_configurator.ini and also use the /etc/nfp_config directory